system-variables.md

```diff
@@ -2381,6 +2381,14 @@ explain select * from t where age=5;
 - Default value: `OFF`
 - This variable sets whether the optimizer rewrites the aggregate functions with `DISTINCT` to the two-level aggregate functions, such as rewriting `SELECT b, COUNT(DISTINCT a) FROM t GROUP BY b` to `SELECT b, COUNT(a) FROM (SELECT b, a FROM t GROUP BY b, a) t GROUP BY b`. When the aggregation column has serious skew and the `DISTINCT` column has many different values, this rewriting can avoid the data skew in the query execution and improve the query performance.
 
+### tidb_opt_three_stage_distinct_agg <span class="version-mark">New in v6.3.0</span>
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
```

Comment:
YES

Reply:
@shichun-0415 There are multiple uses: YES vs Yes.
Which one is preferred?

Reply:
`Yes` is better. I will change `YES` to `Yes`.

Reply:
Fixed in ddb9f3e